### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by erikagayan**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+        "errors": [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "failed" if errors else "passed"
+    }
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [format_single_linter_file(file_path, errors)
```
Comment (LiudmylaKulinchenko) [inline app/main.py:20]: The braces should be located in one of two options: open and start with the text or have a line break between the text. Please, use the same style through the code

File: app/main.py [hunk 2]
```diff
@@ -18,4 +18,5 @@ def format_single_linter_file(file_path: str, errors: list) -> dict:
 
 def format_linter_report(linter_report: dict) -> list:
     return [format_single_linter_file(file_path, errors)
```
Comment (LiudmylaKulinchenko) [inline app/main.py:20]: Fix indentation according to the best practices

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (erikagayan) [reply app/main.py:20]: Sorry, but I still don't understand what to fix. flake8 doesn't write anything. And when I try to write everything in one line, it gives an error: app\main.py:20:80: E501 line too long (104 > 79 characters)

Comment (LiudmylaKulinchenko) [reply app/main.py:20]: It's ok! Look in the checklist from the task. There are a few examples about braces location. Now you have "Bad example", like the next:
```
my_dict = {"greeting": "Good morning, have a nice day!", 
	   "answer": "Good morning, thanks!"
}
```
So your braces should be located in **one** of two options: open and start with the text **or** have a line break between the text. You have both options now

File: app/main.py [hunk 3]
```diff
@@ -1,13 +1,19 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
```
Comment (LiudmylaKulinchenko) [inline app/main.py:2]: Don't forget that your braces should be located in **one** of two options: open and start with the text or have a line break between the text. You have **both** options here and in the next function 🐞

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job! ☀

**Pull request closed without merging**